modules/cloudflare.py

```
@@ -25,12 +25,16 @@ async def getZone(self, zone_id: str) -> dict:
-        async with aiohttp.ClientSession() as session:
-            async with session.get(
-                    url=url,
-                    headers=await self.getHeaders()
-            ) as response:
-                return await response.json()
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.get(
+                        url=url,
+                        headers=await self.getHeaders()
+                ) as response:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)